packer/vhd-image-builder.json

```diff
@@ -193,8 +188,11 @@
             "type": "shell",
             "inline": [
                 "sudo FEATURE_FLAGS={{user `feature_flags`}} BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} /bin/bash -ux /home/packer/install-dependencies.sh",
-                "sudo /bin/bash -eux /home/packer/cleanup-vhd.sh",
-                "sudo rm -rf /home/packer"
+                "sudo rm -f /root/.ssh/authorized_keys",
+                "sudo rm -f /etc/machine-id",
+                "sudo touch /etc/machine-id",
+                "sudo cloud-init clean --logs",
```

Comment:
thinking about this more this technically won't have any effect right now because we are moving the whole /var/log directory (which contains cloud-init logs) as part of the script

Reply:
also we should probably move the var/log dir "clean-up" here

Reply:
it doesn't really make sense for it to be in install-dependencies.sh

Reply:
(or consider removing it all together if we don't want to keep /var/log.vhd for vhd cloud-init logs)